src/carbon_chemistry_coefficients.jl

```diff
@@ -0,0 +1,1366 @@
+end
+
+Base.@kwdef struct Pᴺᴴ⁴ₖ₁{FT}
+    a₀ :: FT = -    0.25444
+    a₁ :: FT = - 6285.33
+    a₂ :: FT =      0.0001635
+    b₀ :: FT =      0.46532
+    b₁ :: FT = -  123.7184
+    b₂ :: FT = -    0.01992
+    b₃ :: FT =      3.17556
+    v₀ :: FT = -   26.43
+    v₁ :: FT =      0.0889
+    v₂ :: FT = -    0.905E-03
+    k₀ :: FT = -    5.03E-03
+    k₁ :: FT =      0.0814E-03
+end
+
+"""
+    Fᴺᴴ⁴ₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴺᴴ⁴ₖ₁)
+
+Return the dissociation constant of ammonium in sea-water [mol/kg-SW], given temperature
+in K, `Θᴷ`, practical salinity, `Sᵖ`, applied pressure, `Δpᵦₐᵣ`, and coefficients, `Pᴺᴴ⁴ₖ₁`.
+
+References: Yao and Millero (1995)
+            Millero (1995) for pressure correction
+pH scale  : SWS
+"""
+@inline function Fᴺᴴ⁴ₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, params = Pᴺᴴ⁴ₖ₁)
+    (; a₀, a₁, a₂, b₀, b₁, b₂, b₃, v₀, v₁, v₂, k₀, k₁ ) = params()
+    ln_knh4_p0 = ( a₀ +
+                     a₁/Θᴷ +
+                        a₂*Θᴷ +
+                     ( b₀ +
+                          b₁/Θᴷ
+                     )*Sᴾ⁰⁵(Sᵖ) +
+                     ( b₂ +
+                          b₃/Θᴷ
+                     )*Sᵖ)
+
+    
+    "Pressure correction for applied pressure /= 0"
+    ln_knh4_pp = (-( v₀ +
+                     v₁*ΘᵒC(Θᴷ) +
+                     v₂*ΘᵒC(Θᴷ)*ΘᵒC(Θᴷ)
+                   )+( 
+                      k₀ +
+                      k₁*ΘᵒC(Θᴷ)
+                     )*(Δpᵦₐᵣ/2.)
+                     )*(Δpᵦₐᵣ/Rₜ(Θᴷ))
+
+    exp( ln_knh4_p0 + ln_knh4_pp )
+end
+
+Base.@kwdef struct Pᴴᶠᵦ₁{FT}
+    a₀ :: FT =     12.641
+    a₁ :: FT = - 1590.2
+    a₂ :: FT = -    1.525
+    v₀ :: FT = -    9.78
+    v₁ :: FT = -    0.0090
+    v₂ :: FT = -    0.942E-03
+    k₀ :: FT = -    3.91e-3
+    k₁ :: FT =      0.054e-3
+end
+
+"""
+    Fᴴᶠᵦ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴴᶠᵦ₁)
+
+Return the association constant of HF in (mol/kg-SW)^-1, , given temperature
+in K, `Θᴷ`, practical salinity, `Sᵖ`, applied pressure, `Δpᵦₐᵣ`, and coefficients, `Pᴴᶠᵦ₁`.
+
+HF <-> H⁺ + F⁻ 
+
+References: Dickson and Riley (1979)
+            Millero (1995) for pressure correction
+pH scale  : free
+Note      : converted here from mol/kg-H2O to mol/kg-SW
+"""
+@inline function Fᴴᶠᵦ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, params = Pᴴᶠᵦ₁)
+    (; a₀, a₁, a₂, v₀, v₁, v₂, k₀, k₁ ) = params()
+    ln_bhf_p0 = ( a₀ +
+                  a₁/Θᴷ +
+                  a₂*sqrt(μₛ(Sᵖ)))
+
+    "Pressure correction for applied pressure /= 0"
+    ln_khf_pp = (-( v₀ +
+                     v₁*ΘᵒC(Θᴷ) +
+                     v₂*ΘᵒC(Θᴷ)*ΘᵒC(Θᴷ)
+                   )+( 
+                      k₀ +
+                      k₁*ΘᵒC(Θᴷ)
+                     )*(Δpᵦₐᵣ/2.)
+                     )*(Δpᵦₐᵣ/Rₜ(Θᴷ))
+    """                
+    Final \beta_HF value
+    --------------------
+     notice that  ln(k_HF(P)) = ln(k_HF(0)) + zln_khf_pp
+            <=>  -ln(\beta_HF(P)) = -ln(\beta_HF(0)) + zln_khf_pp
+            <=>   ln(\beta_HF(P)) =  ln(\beta_HF(0)) - zln_khf_pp
+    """
+    exp(ln_bhf_p0 - ln_khf_pp ) / H₂Oˢʷ(Sᵖ)
```

Comment:
```suggestion
    return exp(ln_bhf_p0 - ln_khf_pp ) / H₂Oˢʷ(Sᵖ)
```
for Julia it doesn't matter, but for humans it's cleaner this way since no need to infer that because it's the last line in the function it's returned

Reply:
remark applies to all function in this file